Dynatrace/dynatrace-operator
# Validate collisions of logmodules and OneAgents in the validation webhook

## Description

It is not possible to deploy the logmodule on the same node that the OneAgent.
To prevent misconfigurations we should add a validation to our validation webhook. 

[Jira](https://dt-rnd.atlassian.net/browse/K8S-10908)


## How can this be tested?

- Deploy a logmodule dynakube
- Deploy a CNFS dynakube => should fail
